File: AtividadesRafael/Capitulo6/Atividade04.c
#include <stdio.h>

struct Livro {
    char titulo[50];
    char autor[50];
    int anoPublicacao;
};

int main() {
    int numeros[5] = {1, 2, 3, 4, 5};
    
    struct Livro livro1 = {"titulo", "autor", 2005};
    
    printf("Título do livro: %s\n", livro1.titulo);
    return 0;
}

File: AtividadesRafael/Capitulo6/Atividade04.java
import java.util.ArrayList;

class Livro {
    String titulo;
    String autor;
    int anoPublicacao;
    
    Livro(String t, String a, int ano){
        titulo = t;
        autor = a;
        anoPublicacao = ano;
    }
}

public class Atividade04 {
    public static void main(String[] args){
        ArrayList<Livro> livros = new ArrayList<>();
        livros.add(new Livro("titulo1", "Naamã", 2000));
        livros.add(new Livro("titulo2", "Naamã2", 2005));
        livros.add(new Livro("titulo3", "Naamã3", 2006));
        
        for(Livro l : livros){
            System.out.println("titulo: " + l.titulo);
        }
    }
}
